docs/weave-net-support.md

```diff
@@ -0,0 +1,67 @@
+# Setup Weave Net CNI monitoring using kube-prometheus
+[Weave Net](https://kubernetes.io/docs/concepts/cluster-administration/networking/#weave-net-from-weaveworks) is a resilient and simple to use CNI for Kubernetes. A well monitored and observed CNI helps in troubleshooting Kubernetes networking problems. [Weave Net](https://www.weave.works/docs/net/latest/concepts/how-it-works/) emits [prometheus metrics](https://www.weave.works/docs/net/latest/tasks/manage/metrics/) for monitoring Weave Net. There are many ways to install Weave Net in your cluster. One of them is using [kops](https://github.com/kubernetes/kops/blob/master/docs/networking.md).
```

Comment:
Ping again about this. It's awkward to call Weave Net a `CNI`. As I mentioned before, CNI is an interface, it is not a concrete thing. Consider something like "CNI provider"

Reply:
pushing

Reply:
done